test_haystack/solr_tests/test_solr_backend.py

```diff
@@ -1639,11 +1639,11 @@ def test_boost(self):
         )
 
         self.assertEqual(
-            [result.id for result in results],
+            sorted([result.id for result in results]),
```

Comment:
```suggestion
            sorted(result.id for result in results),
```
This is one of the ruff rules in https://docs.astral.sh/ruff/rules/#flake8-comprehensions-c4

Reply:
Thanks!